charts/influxdb/values.yaml

```diff
@@ -0,0 +1,311 @@
+## influxdb image version
+## ref: https://hub.docker.com/r/library/influxdb/tags/
+image:
+  repository: "influxdb"
+  tag: "1.7.6-alpine"
+  pullPolicy: IfNotPresent
+  ## If specified, use these secrets to access the images
+  # pullSecrets:
+  #   - registry-secret
+
+## Customize livenessProbe and readinessProbe
+## ref: https://docs.influxdata.com/influxdb/v1.7/tools/api/#ping-http-endpoint
+##
+livenessProbe:
+  probePath: "/ping"
+
+readinessProbe:
+  probePath: "/ping"
+
+startupProbe:
+  enabled: false
+  probePath: "/ping"
+  # failureThreshold: 6
+  # periodSeconds: 5
+
+## Specify a service type
+## NodePort is default
+## ref: http://kubernetes.io/docs/user-guide/services/
+##
+service:
+  ## Add annotations to service
+  # annotations: {}
+  type: ClusterIP
+  ## Add IP Cluster
+  # clusterIP: ""
+  ## Add external IPs that route to one or more cluster nodes
+  # externalIPs: []
+  ## Specify LoadBalancer IP (only allow on some cloud provider)
+  # loadBalancerIP: ""
+  ## Allow source IPs to access on service (if empty, any access allow)
+  # loadBalancerSourceRanges: []
+
+serviceNodePort:
+  name: influxdb-nodeport
+  type: NodePort
+  nodeport: 31333
```

Comment:
Integrate this in service instead. Service can either be ClusterIP or NodePort, not both. Make sure the templates handle this correctly.

Reply:
Also, I recommend using port number 30086 for http service to match the redis & couchdb pattern of using the last 3 port digits.

Reply:
using suggested ports and nodeport specific yaml file was removed